src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -1158,24 +1200,34 @@ namespace OpenMS
       {
         auto& peak_group = deconvolved_spectrum_[i];
         int offset = 0;
-        float cos = getIsotopeCosineAndDetermineIsotopeIndex(peak_group.getMonoMass(), peak_group.getIsotopeIntensities(), offset, avg_, -1, allowed_iso_error_, decoy_flag_);
+        float cos = getIsotopeCosineAndDetermineIsotopeIndex(peak_group.getMonoMass(), peak_group.getIsotopeIntensities(), offset, avg_, -1, allowed_iso_error_, target_dummy_type_);
 
         peak_group.setIsotopeCosine(cos);
 
-        if (cos < .5)
+        if (cos < .45)
         {
           continue;
         }
 
-        peak_group.recruitAllPeaksInSpectrum(deconvolved_spectrum_.getOriginalSpectrum(), tol, avg_, peak_group.getMonoMass() + offset * iso_da_distance_, excluded_peak_mzs_);
+        for(int k = 0;k<5;k++)
```

Comment:
```suggestion
        for (int k = 0; k<5; k++)
```

Reply:
I am reformatting codes now. Please review after the next push :)